public/data/job-openings-data.json

```diff
@@ -674,6 +674,64 @@
           "experience": "1 to 2 Years experience"
         }
       ]
+    },
+    {
+      "id": 16,
+      "companyName": "iView Labs Pvt. Ltd",
+      "location": "Ahmedabad, India",
+      "bio": "iView Labs is a leading cloud-based application development company that provides software solutions to US, Europe, and Australian businesses.",
+      "companyLogoUrl": "https://drive.google.com/file/d/1uMPCQCefDW97WQa22a0Gtj03lZrOqLL1/view?usp=share_link",
```

Comment:
This URL will not work (Company logo will not load), Please refer other jobs object.